test/setup/deployers/helm-with-proxy.ts

```diff
@@ -35,7 +35,8 @@ async function deployKubernetesMonitor(
       `--set image.tag=${imageTag} ` +
       `--set image.pullPolicy=${imagePullPolicy} ` +
       '--set integrationApi=https://kubernetes-upstream.dev.snyk.io ' +
-      '--set https_proxy=http://forwarding-proxy:8080',
+      '--set https_proxy=http://forwarding-proxy:8080 ' +
+      '--set nodeSelector."beta\\.kubernetes\\.io/os"=linux'
```

Comment:
Why set it, is this to validate using it? Can we set it only on one deployer (maybe just helm not helm-with-proxy)?

Reply:
I preferred to keep helm and helm-proxy basically the same, besides the proxy config. wdyt?

Reply:
I'm just conscious we don't have a test where it deploys with the default nodeSelector, so we won't know if something was accidentally broken 🤷

Reply:
that's a very good point, I'll delete it from the helm-with-proxy